superset/viz.py

```diff
@@ -439,7 +440,13 @@ def cache_key(self, query_obj: QueryObjectDict, **extra: Any) -> str:
     def get_payload(self, query_obj: Optional[QueryObjectDict] = None) -> VizPayload:
         """Returns a payload of metadata and data"""
 
-        self.run_extra_queries()
+        try:
+            self.run_extra_queries()
+        except SupersetSecurityException as ex:
```

Comment:
Shouldn't this be captured by the global exception handler?

Reply:
I thought it would be, but it wasn't :/